website/content/docs/concepts/client-count/faq.mdx

```diff
@@ -218,4 +226,22 @@ However, creating a new token across a parent/child namespace boundary could res
+
+Prior to 1.9, customers could create more than one alias from the same auth mount under a single entity. However, we made a fix in Vault 1.9 to prevent this from occuring as a remediation for a security issue.
+
+This fix could potentially impact the number of clients generated for customers. For example, pre Vault 1.9, a customer (using K8s, with many namespaces and services accounts in the cluster) could perform entity management by mapping multiple service accounts under different namespaces under a single entity. By default, each service account will create a unique entity. Post Vault 1.9, they can only map a single K8s cluster to a single K8s auth mount.
+
+To provide further clarity, in post Vault 1.9, if there was an auth mount per namespace, the customer could create an alias for a service account in namespace-1 and an alias for a service account in namespace-2 onto the same entity without issues. However, within namespace-1, which has a single auth mount, if they have a service account-1 and service account-2, both accounts cannot be aliased to a single client.
+
+### Q: How does mount migration impact the client count metric?
+
+In Vault 1.10, we made improvements to the [`sys/remount`](/api-docs/system/remount) API endpoint to simplify the complexities of moving data, such as secret engine and authentication method configuration from one mount to another, within a namespace or across namespaces. This can help with restructuring namespaces and mounts for various reasons, including [migrating mounts](/docs/concepts/mount-migration) from root to other namespaces when transitioning to using namespaces for the first time. To learn more, refer to the [Mount Move](https://learn.hashicorp.com/tutorials/vault/mount-move) tutorial.
+
+When migrating mounts within a namespace, client counts are not impacted.
+
+- For each existing alias on the source mount and its corresponding entity, if the new namespace has an entity corresponding to it, then you can assign a corresponding alias for the mount and the client count will not be impacted. For example, say I am a user `prat` on namespace ns1/ and namespace ns2/, and I have an alias on the source mount in ns1/. After the move operation, I just need an alias added to the `prat` entity in ns2/ for the target mount. This keeps the client counts the same.
+- For the example above, say I do not have an entity `prat` in ns2/. That will need to be created in order to make an alias associating it with the target mount. This increases the client count by 1.
+- Following the above example, let's say after the move operation, the entity `prat` in ns1/ has no other aliases associated with it, i.e., the alias for the source mount was its only alias. At this point, we can clean up and remove the `prat` entity in ns1/. This will decrease the client count by 1.
```

Comment:
```suggestion
- For each existing alias on the source mount and its corresponding entity, if the new namespace has an entity corresponding to it, then you can assign a corresponding alias for the mount and the client count will not be impacted. For example, say I am a user `prat` on namespace `ns1/` and namespace `ns2/`, and I have an alias on the source mount in `ns1/`. After the move operation, I just need an alias added to the `prat` entity in `ns2/` for the target mount. This keeps the client counts the same.
- For the example above, say I do not have an entity `prat` in `ns2/`. That will result in a creation of a default entity in the target mount (`ns2/`). This increases the client count by 1.
- Following the above example, let's say after the move operation, the entity `prat` in `ns1/` has no other aliases associated with it, i.e., the alias for the source mount was its only alias. At this point, we can clean up and remove the `prat` entity in `ns1/`. This will decrease the client count by 1.
```